lib/inspec/transport.rb

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+
+require 'train'
+
+module Inspec
+  class Transport
+    attr_accessor :cache_resources, :conn
```

Comment:
Unless it's necessary, I'd rather we didn't abbreviate variable names. (I know we just did it in the Train platforms PR - i.e. `plat` instead of `platform` - but it still bugs me 🙂 )

Any reason this shouldn't be `connection`?